examples/02-plot/plot-linked.py

```diff
@@ -0,0 +1,50 @@
+"""
+Linked Views in Subplots
+~~~~~~~~~~~~~~~~
```

Comment:
Would you please make sure there are as many `~` as characters in the title line